src/commands/images/selectSignImage.ts

```diff
@@ -0,0 +1,94 @@
+import * as vscode from 'vscode';
+import { logInToDockerCli } from '../registries/logInToDockerCli';
+import { checkCLI } from '../../utils/checkCLI';
+import { execAsync } from '../../utils/execAsync';
+
+const cliName = 'notation';
+const goToNotaryButton = 'Download Notary';
+const NotaryURL = 'https://notaryproject.dev/docs/installation/cli/';
+const errorMessage = `Notary not found: `;
+
+//sorts the key list string into an array
+async function sortOutput(theArray: any ){
+    const dataArray = new Array();
+    var keyChoices = theArray.stdout.split("\n");
+    for (let i = 1; i < keyChoices.length; i++) { 
+      const values = keyChoices[i].trim().split(/ {2,} /); 
+      var itemObject; 
+      for (let j = 0; j < 1; j++) { 
+        const value = values[j];
+        if (value.indexOf('*')> -1){
+            const keyWithoutAsterisk = value.replace('* ', '');
+            itemObject = {label: keyWithoutAsterisk, description: "Current default"};
+        } else {
+            itemObject = {label: value};
+        }
+      }
+      dataArray.push(itemObject);
+    }
+    return dataArray;
+}
+
+//adds precreated azure key to notation
+async function addNewKey() { 
+    var Key_Name = await vscode.window.showInputBox({prompt: "Input the key name"});
+    var AKV_NAME = await vscode.window.showInputBox({prompt: "Input the azure key vault name"});
```

Comment:
Nit. Variable naming should use consistent casing.  For example, see the 
 typescript style guide from google https://google.github.io/styleguide/tsguide.html#identifiers